src/input/keyboard/mod.rs

```diff
@@ -580,6 +580,24 @@ impl<D: SeatHandler + 'static> KeyboardHandle<D> {
         );
     }
 
+    /// Iterate over the keysyms of the currently pressed keys.
+    pub fn with_pressed_keysyms<F>(&self, mut f: F)
```

Comment:
Without seeing the user, it's hard to say but it seems like this shouldn't be `mut`

Reply:
Sure, it needs to be. It's an `FnMut`.